pkg/operator/ceph/controller/spec.go

```diff
@@ -53,6 +53,7 @@ const (
 	daemonTypeLabel                         = "ceph_daemon_type"
 	ExternalMgrAppName                      = "rook-ceph-mgr-external"
 	ServiceExternalMetricName               = "http-external-metrics"
+	livenessProbeTimeoutSeconds       int32 = 1
```

Comment:
This should be 2 now, right?
```suggestion
	livenessProbeTimeoutSeconds       int32 = 2
```